wikimedia/wikipedia-ios
# Add Year in Review entry method visibility logic (Part 1)

**Phabricator:** https://phabricator.wikimedia.org/T376042

### Notes
This PR adds the entry method visibility logic to the profile menu item for Year in Review. It also adds the item to the profile logged out state and localizes the strings.

This task is partially blocked on https://phabricator.wikimedia.org/T376041, https://phabricator.wikimedia.org/T376066 and https://phabricator.wikimedia.org/T376320. I have added todo callouts in the code to make it easy to add that logic in once it's ready. 

### Test Steps
1. Change simulator region to France or Italy.
2. Fresh install the app.
3. Set primary app language to French or Italian.
4. Go to developer settings menu and turn on Year in Review feature flag.
5. Confirm Year in Review option appears on the logged in and logged out profile menu.
6. Confirm tests pass.
